frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -285,8 +286,6 @@ export const EditZing = () => {
       .catch((error) => displayNetworkError(error.message))
   }
```

Comment:
so uh, what's going on here?

Reply:
I believe Changyuan added a catch for any general errors

Reply:
See https://github.com/cornell-dti/zing-lsc/pull/65#issuecomment-1153049016. It's better to update the timestamp locally to approximate what was changed on the backend instead of making a second network request getting all of the data again. Like when dragging and dropping students between groups, the groups are updated locally and the request is made to save. It doesn't make the request and then reload all the data again. But @notwz just merged in that PR right after making the commits lol https://github.com/cornell-dti/zing-lsc/blob/e7a9ee1f665d6acd31eb9e01c77397c5b94e81af/frontend/src/modules/EditZing/Components/EditZing.tsx#L128-L150